### PR TITLE
(grc_conversion) continued EpiDoc and CTS conversion work

### LINKED ISSUE #1630: tlg0016.tlg001.perseus-eng2: 1.74.5 is missing and mistakenly marked as 1.74.6
I am surprised this error has been around so long, assuming it is what I see in the text.

### PR DESCRIPTION
**Pull request by lcerrato**
work on #1399 
fix #1630

### CODE REVIEW

File: data/tlg0027/tlg003/__cts__.xml [hunk 1]
```diff
@@ -4,7 +4,12 @@
     
     <ti:edition urn="urn:cts:greekLit:tlg0027.tlg003.perseus-grc2" workUrn="urn:cts:greekLit:tlg0027.tlg003" xml:lang="grc">
         <ti:label xml:lang="grc">Περὶ τῆς πρὸς Λακεδαιμονίους εἰρήνης</ti:label>
-        <ti:description xml:lang="mul">Andocides. Minor Attic Orators, Vol. 1. Maidment, K. J., editor. London: William Heinemann, Ltd.; Cambridge, MA: Harvard University Press, 1941.</ti:description>
```
Comment (AlisonBabeu) [inline data/tlg0027/tlg003/__cts__.xml:7]: You have changed editor to translator here again.

File: data/tlg0027/tlg004/__cts__.xml [hunk 2]
```diff
@@ -4,7 +4,12 @@
     
     <ti:edition urn="urn:cts:greekLit:tlg0027.tlg004.perseus-grc2" workUrn="urn:cts:greekLit:tlg0027.tlg004" xml:lang="grc">
         <ti:label xml:lang="grc">Κατὰ Ἀλκιβιάδου</ti:label>
-        <ti:description xml:lang="mul">Andocides. Minor Attic Orators, Vol. 1. Maidment, K. J., editor. London: William Heinemann, Ltd.; Cambridge, MA: Harvard University Press, 1941.</ti:description>
+        <ti:description xml:lang="mul">Andocides. Minor Attic Orators, Vol. 1. Maidment, Kenneth John, translator. London: William Heinemann, Ltd.; Cambridge, MA: Harvard University Press, 1941 (1960 printing).</ti:description>
```
Comment (AlisonBabeu) [inline data/tlg0027/tlg004/__cts__.xml:7]: This should be editor not translator.

**Review by AlisonBabeu — CHANGES_REQUESTED** [review]
hi @lcerrato just one minor change in the cts_xml files!

**Comment by lcerrato** [comment]
@AlisonBabeu  I think I have them all?

**Review by AlisonBabeu — APPROVED** [review]
hi @lcerrato you definitely have them all!

**Pull request merged**